docker-images/jenkins/Dockerfile

```diff
@@ -25,24 +25,15 @@ curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add -;\
 add-apt-repository  "deb [arch=amd64] https://download.docker.com/linux/debian  $(lsb_release -cs) stable";\
 apt-get update && apt-get install docker-ce -y
 
-# Installing kubectl from Docker
-RUN curl -fsSL https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -;\
-touch /etc/apt/sources.list.d/kubernetes.list;\
-echo "deb http://apt.kubernetes.io/ kubernetes-xenial main" | tee -a /etc/apt/sources.list.d/kubernetes.list;\
-apt-get update && apt-get install -y kubectl
-
-# Installing utils for reverese DNS lookup
-RUN apt-get install knot-dnsutils -y
-RUN kdig
 # Installing maven
 RUN apt-get install maven -y
-RUN mvn -v
 
-# Install helm
+# Install Helm
 RUN curl https://raw.githubusercontent.com/helm/helm/master/scripts/get > get_helm.sh
```

Comment:
@Aaquiff shall we combine all these individual Dockefile `RUN` commands to a single `RUN` command. Each Dockerfile command effectively adds a new image layer.